shuup/discounts/models/_discounts.py

```diff
@@ -63,23 +60,19 @@ def available(self, shop=None):
             & (
                 Q(happy_hours__time_ranges__isnull=True)
                 | Q(
+                    # TODO: check whether there is a bug here when a time
+                    # advances a day in a UTC day, e.g: 11pm - 01am
```

Comment:
I think this might be okay as long as everything is tz-aware:
```>>> import pytz
>>> from django.utils.timezone import now, localtime
>>> # UTC first, to make it earlier
>>> utc = now()
>>> utc.hour 
16
>>> # Eastern Daylight Time is later, though the hour is a smaller number
>>> edt = localtime(timezone=pytz.timezone("America/New_York"))
>>> edt.hour 
12
>>> utc < edt
True
```